### DISCUSSION TITLE
Import Contacts

### DISCUSSION THREAD

**Discussion by MM502** [juha-h/baresip-studio#338] [Q&A]
I am new to baresip and can't figure out how to import contacts. Baresip has permission to access the Android address book but all I get is"The Test Call."
Many thanks!

**Comment by MM502** [comment]
My mistake! I selected contacts "both" in settings but did not press the check mark (used back arrow instead).
Working fine now!

**Comment by juha-h** [comment]
MM502 writes:

> I am new to baresip and can't figure out how to import
> contacts. Baresip has permission to access the Android address book
> but all I get is"The Test Call."

In Settings, choose Android contacts.
